pkgs/os-specific/linux/kernel/common-config.nix

```diff
@@ -244,6 +244,7 @@ let
       SND_HDA_RECONFIG    = yes; # Support reconfiguration of jack functions
       # Support configuring jack functions via fw mechanism at boot
       SND_HDA_PATCH_LOADER = yes;
+      SND_HDA_CODEC_CA0132_DSP = yes; # Enable DSP firmware loading on Creative Soundblaster Z/Zx/ZxR/Recon
```

Comment:
This should link to some issue / commit upstream, so it's possible to check when we can drop it.

Reply:
Looking at Takshis tree this is staged for next, which will mean this will be in 5.8. I'll amend.